Scripts/build/Jenkins/Jenkinsfile

```diff
@@ -419,18 +469,42 @@ def ExportTestResults(Map options, String platform, String type, String workspac
     }
 }
 
-def ExportTestScreenshots(Map options, String workspace, String platformName, String jobName, Map params) {
+def ExportTestScreenshots(Map options, String branchName, String platformName, String jobName, String workspace, Map params) {
     catchError(message: "Error exporting test screenshots (this won't fail the build)", buildResult: 'SUCCESS', stageResult: 'FAILURE') {
-        def screenshotsFolder = '${workspace}/${PROJECT_REPOSITORY_NAME}/user/Scripts/Screenshots'
-        def s3Uploader = '${workspace}/${ENGINE_REPOSITORY_NAME}/scripts/build/tools/upload_to_s3.py'
-        def command = '${options.PYTHON_DIR}/python.cmd -u ${s3Uploader} --base_dir ${screenshotsFolder} ' +
-                      '--file_regex "(.*png$|.*ppm$)" --bucket ${env.TEST_SCREENSHOT_BUCKET} '
-                      '--search_subdirectories True --key_prefix ${branchName}_${env.BUILD_NUMBER}'
-        bat label: "Uploading test screenshots for ${jobName}",
-            script: command
+        dir("${workspace}/${ENGINE_REPOSITORY_NAME}") {
+            def screenshotsFolder = "AutomatedTesting/user/PythonTests/Automated/Screenshots"
```

Comment:
This looks backwards.
The screenshotsFolder on the left is the one that should work.

Reply:
I made a mistake in committing to this branch. I've closed this PR in favor of the correct changes